components/camera/transformpipeline/depth_edges.go

```diff
@@ -23,13 +22,13 @@ type depthEdgesConfig struct {
 
 // depthEdgesSource applies a Canny Edge Detector to the depth map.
 type depthEdgesSource struct {
-	stream     gostream.VideoStream
+	src        camera.VideoSource
```

Comment:
Why can't this be a camera?

Reply:
https://github.com/viamrobotics/rdk/pull/4487#discussion_r1842930728